src/irmin-pack/inode.ml

```diff
@@ -106,6 +106,15 @@ struct
       |~ case1 "Tree" tree_t (fun t -> Tree t)
       |> sealv
 
+    let v_t =
+      if not Conf.prefix_unsable_pre_hash then v_t
+      else
+        let pre_hash : v Irmin.Type.encode_bin =
+          Irmin.Type.(
+            pre_hash (map (pair char v_t) (fun (_, s) -> s) (fun s -> ('i', s))))
```

Comment:
you can probably extract this function in `import.ml` and call it `pre_hash_with_prefix` or something similar.

Reply:
👍